content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+   appear on the pop-up.
+
+Universal fields
+~~~~~~~~~~~~~~~~
+
+The following fields appear for all offers sent to applicants regardless of localization settings.
+
+- :guilabel:`Offer Template`: The template currently being used to populate the :guilabel:`Generate
+  a Simulation Link` pop-up. To modify the template, click the :guilabel:`External link` icon to the
+  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
+- :guilabel:`Job Title`: The name of the position being offered to the applicant.
+- :guilabel:`Contract Start Date`: The date the contract takes effect. The default date is the
+  current date. To modify the date, click the drop-down to reveal the calendar module. Use the
+  :guilabel:`< (left)` and :guilabel:`> (right)` arrows on either side of the month to navigate to
+  the desired month, then click on the :guilabel:`day` to select the date.
+- :guilabel:`Yearly Cost`: The annual salary being offered.
+- :guilabel:`Offer Link`: This link takes the applicant to a webpage where they can enter their
+  personal information, which will be imported to their employee record when created. If applicable,
+  the applicant can modify their salary package (this option is not available for all
+  localizations). Lastly, this is where the applicant accepts the offer by clicking
+  :guilabel:`Review Contract & Sign` to accept the contract and sign it.
+
+Optional fields
+~~~~~~~~~~~~~~~
+
+The following fields will only appear if specific localization settings are selected, and/or certain
+applications are installed.
+
+- :guilabel:`Job Position`: The official title of the job being offered to the applicant, from the
+  :guilabel:`Job Position` card. To modify the name, click the :guilabel:`External link` icon to the
+  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
+- :guilabel:`Department`: The department the job position falls under. To modify the department,
+  click the :guilabel:`External link` icon to the right of the drop-down arrow. Make any changes,
+  then click :guilabel:`Save`.
+- :guilabel:`Force New Cars List`: Check this box to offer a new vehicle to the applicant.
+- :guilabel:`Default Vehicle`: Using the drop-down menu, select the vehicle that will be assigned
+  to the applicant.
+- :guilabel:`Canteen Cost`: Enter the amount that is offered to the applicant for the cafeteria.
+- :guilabel:`Contract Type`: Using the drop-down menu, select the type of contract that is being
+  offered.
+
+.. image:: recruitment/generate-link.png
+   :align: center
+   :alt: Format the offer to send to the applicant. Make any modifications before sending.
+
+Send offer
+~~~~~~~~~~
+
+To send the offer to the applicant, click :guilabel:`Send`. An email pop-up appears, using the
+:guilabel:`Applicant: Your Salary Package` email template. The :guilabel:`Recipients`,
+:guilabel:`Subject`, and email body are pre-populated based on the email template. If any
```

Comment:
A save contact option pop-up happened first!

Reply:
True- if they don't have an email on file. I will add that!